content/workshops/ansible_tower/exercise3.0.adoc

```diff
@@ -56,7 +56,7 @@ Complete the project form, using the following entries:
 |DESCRIPTION|STIG Role Playbook
 |ORGANIZATION|Default
 |SCM TYPE|Git
-|SCM URL| https://github.com/ajacocks/rhel7_disa_stig.git
+|SCM URL| https://github.com/RedHatOfficial/ansible-role-rhel7-stig.git
```

Comment:
Wasn't sure if the workshop was intended to be ran from @ajacocks 's fork or not. Here's the upstream repo URL. Can revert as needed.